webpack.config.ts

```diff
@@ -535,7 +535,22 @@ if (
     }
   }
 
+  // Try and load certificates from mkcert if available. Use $ yarn mkcert-localhost
+  const certPath = path.join(__dirname, 'config');
+  const useHttps = fs.existsSync(path.join(certPath, 'localhost.pem'));
+
+  const httpsOptions = !useHttps
+    ? {}
+    : {
+        key: fs.readFileSync(path.join(certPath, 'localhost-key.pem')),
+        cert: fs.readFileSync(path.join(certPath, 'localhost.pem')),
+      };
+
   appConfig.devServer = {
+    server: {
+      type: useHttps ? 'https' : 'http',
```

Comment:
As I noted in the PR description, maybe we should just always use https, and update the develop docs to always include installing and running `mkcert`?